roles/kalite/tasks/install.yml

```diff
@@ -17,9 +17,13 @@
       - python-setuptools    # Provides setuptools-44 on recent OS's (last version compatible with python2)
       - virtualenv           # Drags in 'python3-virtualenv' which in turn drags in 'python3-pip' -- for Ansible module 'pip' when used with 'virtualenv_command: /usr/bin/virtualenv' and 'virtualenv_python: python2.7' -- compare package 'python3-venv' used by roles {calibre-web, jupyterhub, lokole}
     state: present
-  #when: not (is_debian_9 or is_debian_10 or is_ubuntu_16 or is_ubuntu_17 or is_ubuntu_18 or is_ubuntu_19)
+  when: not (is_debian_12 or is_ubuntu_2304)
```

Comment:
```suggestion
  when: is_debian and os_ver is version('debian-12', '<') or is_ubuntu and os_ver is version('ubuntu-2304', '<')
```

Reply:
same deal as above, let move away from OS.yml when os_ver is in already declared in the environment

Reply:
same issue

Reply:
Parsing `os_ver` strings won't always be pretty, even while standard Ansible syntax can certainly do that.

Removing all `<OS>.yml` files from /opt/iiab/iiab/vars while doable, might be premature.

i.e. possibly way too hairy.